CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 1.0.0 / 2022-03-11
+
```

Comment:
Need to add a breaking change section here like https://github.com/kubernetes-sigs/kubebuilder/releases

### Breaking Changes:
- All Fluent Bit CRDs has been changed from namespaced scope to cluster scope
- Fluentd CRDs has been added

### Features
- xx

### Enhancements

### Fixes